src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -291,8 +292,7 @@
             "ActiveModeThreshold",
             "RegisteredClients",
             "ICDCounter",
-            "ClientsSupportedPerFabric",
-            "MaximumCheckInBackOff"
```

Comment:
What does MaximumCheckInBackOff have to do with TLSCertificateManagement?

Reply:
It looks like someone manually modified this instead of using Alchemy, resulting in an invalid state; `MaximumCheckInBackOff` is tagged as `apiMaturity="provisional"` so it should not be here

Reply:
I'm going to dig in to whether this was an issue with the spec or with Alchemy (or user error on my part w/ unclean client), manually fixed